src/transaction/transaction.ts

```diff
@@ -622,11 +622,9 @@ export const sendAtomicTransactionComposer = async function (atcSend: AtomicTran
 
     if (Config.debug && Config.projectRoot && Config.traceAll) {
```

Comment:
Should we still be checking for `Config.projectRoot`? If I understand correctly, it won't be set by us anymore.